mosqagent.c

```diff
@@ -15,6 +15,7 @@
 #include <mosquitto.h>
 
 #include "mosqhelper.h"
+#include "mqtt-build.h"
```

Comment:
this include file should be named `mqtta-build.h`

Reply:
Funny, I would have named it the same with the directory restructuring. ^^